Q:

Raspbian ROS Indigo - Rosserial_xbee error

Hello, I've been trying out the Xbee network tutorial from the wiki: http://wiki.ros.org/rosserial_xbee/Tutorials/Example%20Network
My setup: Raspberry Pi with Xbee Series 1 and Arduino Uno with Wireless SD shield that has a Xbee Series 1 on top of that.
I have configured the Xbee's with XCTU and the xbee is connected to the AMA0 port
When I try to run the command: rosrun rosserial_xbee xbee_network.py /dev/ttyAMA0 1 I can see the Arduino flashing as if it is sending the data. When i open Arduino's port monitor i can also see that it is sending the data. So i guess the Xbee connection isn't the problem here?
I get the following error:
Traceback (most recent call last): 
File "/usr/lib/python2.7/threading.py", line 552, in __bootstrap_inner
   self.run()
File "/usr/lib/python2.7/threading.py", line 505, in run 
   self.__target(*self.__args, **self.__kwargs)
File "/home/pi/catkin_ws/src/rosserial/rosserial_python/src/rosserial_python/SerialClient.py", line 434, in run 
   if self.port.inWaiting() < 1:
AttributeError: FakeSerial instance has no attribute 'inWaiting'

I found only one other person struggling with the same problem here; http://answers.ros.org/question/187443/xbee-network-error/
But no obvious fix was found?
The marked answer suggests that installing ROS hydro rosserial fixes the problem. Can that actually work?

Originally posted by rip24 on ROS Answers with karma: 13 on 2015-03-31
Post score: 1

A:

I posted an issue on the github rosserial site
I have marked the Issue resolved
Please visit the link above to see the solution.

Originally posted by rmolin88 with karma: 26 on 2015-04-03
This answer was ACCEPTED on the original site
Post score: 1

Original comments
Comment by rip24 on 2015-04-03:
Thanks for the info, atleast now i know that this is an issue for everyone =)

